content/content.go

```diff
@@ -216,6 +216,30 @@ func (s *RulesWithContentStorage) GetExternalRuleIDs() []ctypes.RuleID {
 	return s.externalRuleIDs
 }
 
+// GetRuleSeverities returns a map of rule IDs and their severity (total risk)
+// along with a list of unique severities
+func (s *RulesWithContentStorage) GetRuleSeverities() (
+	severityMap map[ctypes.RuleID]int,
+	uniqueSeverities []int,
+) {
+	s.Lock()
+	defer s.Unlock()
+
+	severityMap = make(map[ctypes.RuleID]int)
+	uniqueMap := make(map[int]bool)
```

Comment:
`map[interface{}]interface{}` is usually used as a set

Reply:
thank you, we know (need) the type of the keys, so I changed it to `map[int]interface{}`